vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/GridViewClickListenersPage.java

```diff
@@ -37,6 +39,12 @@ private void createClickListener() {
         grid.setItems(getItems());
         grid.addColumn(Person::getFirstName).setHeader("Name");
         grid.addColumn(Person::getAge).setHeader("Age");
+        grid.addColumn(new ComponentRenderer<>(person -> {
+            NativeButton button = new NativeButton("Action");
+            button.getElement().executeJs(
+                    "$0.addEventListener('click', e => e.preventDefault())");
```

Comment:
The event gets blocked by something else because the test case passes even without this event listener in place.

Looks like it's because `button` is listed [here](https://github.com/vaadin/web-components/blob/1f9603fcba7d56f16b49e211339f802cd51ae7c0/packages/grid/src/vaadin-grid-active-item-mixin.js#L111)

Reply:
Good catch, thanks. Updated to use `span` instead.